src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json

```diff
@@ -0,0 +1,119 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions",
+    "@type": "hydra:Collection",
+    "title": "Aged 16 to 64 years level 3 or above qualifications",
+    "editions": [
+      {
+        "@id": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions/2023-05",
+        "@type": "dcat:Dataset",
+        "in_series": "https://staging.idpd.uk/datasets/age-16-to-64-years-level",
+        "identifier": "2023-05",
+        "title": "Aged 16 to 64 years level 3 or above qualifications",
+        "summary": "Percentage of the population aged 16 to 64 with National Vocational Qualification (NVQ) level 3+ qualifications, Great Britain (Northern Ireland national figure included), 2004 to 2021",
+        "description": "This dataset shows the percentage of the population aged 16 to 64 with National Vocational Qualification (NVQ) level 3+ qualifications, in Great Britain (Northern Ireland national figure included), for 2004 - 2021.",
+        "publisher": "https://staging.idpd.uk/publishers/department-for-education",
+        "creator": "https://staging.idpd.uk/publishers/department-for-education",
+        "contact_point": {
+          "name": "ofsted",
+          "email": "mailto:informationrequest@ofsted.gov.uk"
+        },
+        "topics": [
+          "https://staging.idpd.uk/topics/children-education-and-skills"],
+        "frequency": "annual",
+        "keywords": [
+          "annual-population",
+          "annual-survey",
+          "national-vocational",
+          "national-qualification",
+          "subnational",
+          "local-authority",
+          "region"
+        ],
+        "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+        "issued": "2023-05-23T10:17:47Z",
+        "modified": "2023-08-27T13:10:00Z",
+        "spatial_resolution":["E06", "E07", "E08", "E09", "E11"],
+        "spatial_coverage": "E92000001",
+        "temporal_resolution": ["P1Y"],
+        "temporal_coverage": {
+          "start": "2004-01-01T12:00:00Z",
+          "end": "2021-12-31T12:00:00Z"
+        },
+        "versions_url": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions/2023-05/versions",
+        "versions": [
+          {
+            "@id": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions/2023-05/versions/1",
+            "issued": "2023-05-23T10:17:47Z"
+          }
+        ],
+        "next_release": "2024-05-01T09:30:00Z",
+        "table_schema": {
+          "columns": [
+            {
+              "name": "period",
+              "datatype": "datetime",
+              "titles": "Gregorian Time Interval",
+              "description": "The time period associated with the observation."
+            },
+            {
+              "name": "areacd",
```

Comment:
Please check this column mapping (title, etc) with the `aged-16-to-64-years-level-3-or-above-qualifications.json `on https://github.com/GSS-Cogs/family-ess

Also check the column mapping for other dimensions, as well.

Some columns were set to `false`. In that case, we don't need to map those columns